readme.txt

```diff
@@ -182,7 +182,7 @@ Check out our extensive [knowledgebase](https://evnt.is/18wm) for articles on us
 
 = [5.1.6] TBD =
 
-
+* Tweak - Added $ticket_id to 'tribe_events_tickets_metabox_edit_ajax_advanced'. [ETP-111]
```

Comment:
```suggestion
* Tweak - Added `$ticket_id` parameter to the `tribe_events_tickets_metabox_edit_ajax_advanced` filter. [ETP-111]
```
Maybe we can add some more info of the change. We normally tend to use backticks when using variable names and hooks in the changelog entries, as they normally are use in different places to represent code format.